Doc/#1209 - #1250 - Add documentation page for each application and page template (#1263)
Avaiga/taipy-doc: docs/userman/ecosystem/templates/applications/sdm_app.md

> Taipy scenario management application template lets you create a custom Taipy
> application with scenario management capabilities, which utilizes
> [a scenario selector](./../../../refmans/gui/viselements/corelements/scenario_selector.md) to allow
> creating, managing, and running scenarios directly from the GUI page.

Reviewer: ```suggestion
Taipy's scenario management application template lets you scaffold a custom Taipy
application using scenario management capabilities. It uses
[a scenario selector](./../../../refmans/gui/viselements/corelements/scenario_selector.md)
to allow end users to create, manage, and run scenarios directly from the GUI page.
```

> 

Reviewer: ```suggestion
# Description
...
```

> ## Create a Taipy application from the scenario management template

Reviewer: Same remark. It shold be a first-level section.
```suggestion
# Create a Taipy application from the scenario management template
```
